header.php

```diff
@@ -10,8 +10,6 @@
 <html <?php language_attributes(); ?>>
   <head>
     <meta charset="<?php bloginfo('charset'); ?>">
-    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
-    <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
```

Comment:
Is there a reason why you removed these scripts?

Reply:
The Quicksand font is already being loaded through bootstrap.css and jquery 3.3.1 is already loaded through enqueue.php, so it seemed these scripts were being loaded twice.